scopes/ui-foundation/react-router/ui/nav-link/nav-link.tsx

```diff
@@ -5,7 +5,7 @@ import { LinkAnchor } from '@teambit/ui-foundation.ui.react-router.link-anchor';
 
 export { NavLinkProps };
 
-// React Router equivalent of an <a/> tag, with added styles when href matches the current url.
+/** Adapter between React router's Nave and our isomorphic link components. Learn more [Here](https://bit.dev/teambit/base-ui/routing/routing-provider) */
```

Comment:
Nave?

Reply:
also consider moving to the docs file. not sure we support links in the subtitle

Reply:
```suggestion
/** Adapter between React router's Nav and our isomorphic link components. Learn more [Here](https://bit.dev/teambit/base-ui/routing/routing-provider) */
```

yes I'm waiting for the day it will work haha